crd-convert/src/main/java/io/strimzi/kafka/crd/convert/cli/ClearCommand.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.cli;
+
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+@CommandLine.Command(name = "clear", aliases = {"cls"}, description = "Clear console")
```

Comment:
I don't understand what this does or why we need it. A class javadoc which explained at much would be good.

Reply:
@alesj